vsgenstubs4/_vapoursynth.part.pyi

```diff
@@ -391,17 +400,27 @@ class Core:
     num_threads: int
     max_cache_size: int
 
+    def plugins(self) -> typing.Iterator[Plugin]: ...
+    # get_plugins is deprecated
     def get_plugins(self) -> typing.Dict[str, _PluginMeta]: ...
     def list_functions(self) -> str: ...
 
-    def query_video_format(self, color_family: ColorFamily, sample_type: SampleType, bits_per_sample: int, subsampling_w: int, subsampling_h: int) -> VideoFormat: ...
+    def query_video_format(self, color_family: ColorFamily, sample_type: SampleType, bits_per_sample: int, subsampling_w: int = 0, subsampling_h: int = 0) -> VideoFormat: ...
+    # register_format is deprecated
+    def register_format(self, color_family: ColorFamily, sample_type: SampleType, bits_per_sample: int, subsampling_w: int, subsampling_h: int) -> VideoFormat: ...
     def get_video_format(self, id: typing.Union[VideoFormat, int, PresetFormat]) -> VideoFormat: ...
+    # get_format is deprecated
+    def get_format(self, id: typing.Union[VideoFormat, int, PresetFormat]) -> VideoFormat: ...
+    def log_message(self, message_type: MessageType, message: str) -> None: ...
+    def add_log_handler(self, handler_func: typing.Optional[typing.Callable[[MessageType, str], None]]) -> None: ...
+    def remove_log_handler(self, handle) -> None: ...
 
     def version(self) -> str: ...
     def version_number(self) -> int: ...
 
-
-def get_core(threads: typing.Optional[int]=None, add_cache: typing.Optional[bool]=None) -> Core: ...
+    def __dir__(self) -> typing.List[str]: ...
+    def __str__(self) -> str: ...
```

Comment:
It seems a bit superfluous to say that __str__ returns a string, I don't think you should add every single method to this file